src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunnerFactory.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Stryker.Core.Initialisation;
+using Stryker.Core.Options;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Stryker.Core.TestRunners.VsTest
+{
+    public class VsTestRunnerFactory : ITestRunner
```

Comment:
I find the name `VsTestRunnerFactory` a bit confusing. Maybe `VsTestRunnerObjectPool` covers the subject better?